src/components/navbar/Navbar.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+
+export const Navbar = () => {
+  return (
+    <header>
+    <div>
+      <h1 className="header-logo">B-Quiz</h1>
+      <small className="slogan">Test your creativity skills</small>
+    </div>
+    <div className="mr-auto page-link-wrapper font3">
+      <ul className="style-list-none">
+        <li className="inline-block">
+          <a href="index.html" className="page-link decor-none active-item"
+            >Home</a
+          >
+        </li>
+        <li className="inline-block">
+          <a href="dashboard.html" className="page-link decor-none">dashboard</a>
```

Comment:
You can replace <a> tags with Link from React Router

Reply:
Yep, Will be doing when I work on that page